packages/autocomplete/src/autocomplete.vue

```diff
@@ -18,7 +18,7 @@
       @keydown.up.native.prevent="highlight(highlightedIndex - 1)"
       @keydown.down.native.prevent="highlight(highlightedIndex + 1)"
       @keydown.enter.native="handleKeyEnter"
-      @keydown.native.tab="close"
+      @keydown.native.tab="handleKeyEnter"
```

Comment:
Is this an intended functional change, i.e. tab to select the currently highlighted value? And if so shouldn't it also blur the field (as opposed to Enter)? (This also seems like something that could be a configurable prop, though if we're ok with not preserving existing behavior then it wouldn't really be necessary.)